packages/frontend/src/components/nft/NFTTransferModal.js

```diff
@@ -0,0 +1,381 @@
+import FormButton from '../common/FormButton';
+import Modal from '../common/modal/Modal';
+import ModalFooter from '../common/modal/ModalFooter';
+import SafeTranslate from '../SafeTranslate';
+import ReceiverInputWithLabel from '../send/components/ReceiverInputWithLabel';
+import AvatarSuccessIcon from '../svg/AvatarSuccessIcon';
+import EstimatedFees from '../transfer/EstimatedFees';
+
+const StyledContainer = styled.div`
+    &&& {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        padding: 0;
+
+        img {
+            width: 100%  ;
+            border-radius: 8px;
+            margin: 16px 0;
+        }
+
+        .transfer-txt {
+            margin-top: 16px;
+        }
+
+        .confirm-txt {
+            margin-bottom: 4px; 
+        }
+
+        .confirm-img {
+            max-width: 272px;
+        }
+
+        .confirm-nft-card {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            position: relative;
+            width: 100%;
+            margin-top: 16px;
+            background: #FFFFFF;
+            box-shadow: 0px 45px 56px rgba(0, 0, 0, 0.07), 0px 10.0513px 12.5083px rgba(0, 0, 0, 0.0417275), 0px 2.99255px 3.72406px rgba(0, 0, 0, 0.0282725);
+            border-radius: 8px;
+
+            .from-box {
+                position: relative;
+                width: 100%;
+                height: 74px;
+                border-top: 1px solid #F0F0F1;
+            }
+
+            .to-box {
+                position: relative;
+                width: 100%;
+                height: 74px;
+                border-top: 1px solid #F0F0F1;
+            }
+
+        }
+
+        .confirm-txt {
+            font-size: 14px;
+            line-height: 150%;
+            color: #72727A;
+            margin-left: 16px;
+        }
+
+
+        .h-right {
+            position: absolute;
+            right: 16px;
+            text-align: right;
+        }
+
+        .estimated-fees {
+            width: 100%;
+            margin-top: 20px;
+        }
+
+        .account-id {
+            font-weight: bold;
+            font-size: 14px;
+            line-height: 150%;
+            text-align: right;
+            color: #272729;
+        }
+
+        h3 {
+            font-weight: 900;
+            font-size: 20px;
+            line-height: 130%;
+            text-align: center;
+            color: #24272A;
+        }
+
+        .success {
+            > p {
+                font-weight: 500;
+                font-size: 20px;
+                line-height: 150%;
+                text-align: center;
+                color: #3F4045;
+                margin-bottom: 0px;
+            }
+        }
+
+        p {
+            font-weight: 500;
+            font-size: 14px;
+            line-height: 150%;
+            text-align: center;
+            color: #72727A;
+        }
+
+        .next-btn {
+            margin-left: 44px  ;
+        }
```

Comment:
```suggestion

```

Reply:
Can we remove this? Seems to add unnecessary extra space on the left side of the button?

<img width="580" alt="Screen Shot 2022-01-03 at 2 57 16 PM" src="https://user-images.githubusercontent.com/24921205/147989324-ca808393-aad8-4a7f-8297-7dde834e0519.png">